doc/Manuals/ug-ss_x-road_6_security_server_user_guide.md

```diff
@@ -3322,4 +3326,29 @@ eab-credentials:
           kid: kid123
           mac-key: goodlongsecretwordthatisnotshort
 
-```
\ No newline at end of file
+```
+
+## 25 Migrating to EC based Authentication and Signing certificates
+
+### 25.1 Steps to enable EC based certificates
+
+Since version 7.6.0 Security Server supports ECDSA based Authentication and Signing certificates. By default, both authentication and signing certificates will use RSA algorithm as in previous versions. EC algorithm can be enabled separately for authentication and/or signing keys so migration can be done gradual way.
+The instructions how to start using EC based certificates are listed below.
```

Comment:
Consider updating this:

> Since version 7.6.0 Security Server supports ECDSA based authentication and signing keys. By default, both authentication and signing keys use the RSA algorithm as in previous versions. The EC algorithm can be enabled separately for authentication and/or signing keys so migration can be done in a gradual way.
The instructions on how to start using EC based keys are listed below.